core/src/main/java/org/aerogear/mobile/core/http/OkHttpCertificatePinningParser.java

```diff
@@ -0,0 +1,25 @@
+package org.aerogear.mobile.core.http;
+
+import java.util.List;
+
+import org.aerogear.mobile.core.configuration.https.CertificatePinningEntry;
+
+import okhttp3.CertificatePinner;
+
+public class OkHttpCertificatePinningParser implements HttpCertificatePinningParser {
+
+    private final List<CertificatePinningEntry> pinningConfig;
+
+    public OkHttpCertificatePinningParser(final List<CertificatePinningEntry> pinningConfig) {
+        this.pinningConfig = pinningConfig;
```

Comment:
The list should be copied to avoid caller having a reference to a mutable internal